azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/AppServiceUtils.java

```diff
@@ -126,4 +128,10 @@ public static DockerImageType getDockerImageType(final String imageName, final S
             return isPrivate ? DockerImageType.PRIVATE_DOCKER_HUB : DockerImageType.PUBLIC_DOCKER_HUB;
         }
     }
+
+    public static <T extends WebAppBase> T findAppServiceInPagedList(PagedList<T> list, String resourceGroup, String name) {
+        final Iterator<T> iterator = list.listIterator();
```

Comment:
need to check list for null and resourceGroup/name for non-empty

Reply:
Added